### PR TITLE
[TD-12501]<docs>: add upgrade/uninstall section for TDinsight

### PR DESCRIPTION
**Pull request by zitsen**

### CODE REVIEW

File: dashboards/TDinsight.md [hunk 1]
```diff
@@ -349,6 +349,22 @@ Here's the metrics list:
 
 Currently only report login count per minute.
 
+## Upgrade
+
+Re install with `TDinsight.sh` will upgrade Grafana plugin and TDinsight dashboard.
```
Comment (sangshuduo) [inline dashboards/TDinsight.md:354]: Reinstall

Comment (sangshuduo) [reply dashboards/TDinsight.md]: TDinsight dashboard(s)

Comment (zitsen) [reply dashboards/TDinsight.md]: I prefer to use dashboard as it specificly only one.

File: dashboards/TDinsight.md [hunk 2]
```diff
@@ -349,6 +349,22 @@ Here's the metrics list:
...
+
+In other cases, users should install new TDengine data source plugin, then remove and install new TDinsight dashboard.
```
Comment (sangshuduo) [inline dashboards/TDinsight.md:356]: In other cases, users should install the new TDengine data source plugin first, then remove and install the new TDinsight dashboard.

Is it better?

File: dashboards/TDinsight.md [hunk 3]
```diff
@@ -349,6 +349,22 @@ Here's the metrics list:
...
+
+## Uninstall
+
+With `TDinsight.sh`, just run `TDinsight.sh -R` to cleanup all installed components.
+
+Follow the next steps to cleanup if you have installed TDinsight manually.
```
Comment (sangshuduo) [inline dashboards/TDinsight.md:362]: clean up

**Review by sangshuduo — COMMENTED** [review]
commented

File: dashboards/TDinsight.md (hunk 2 above)
Comment (zitsen) [reply dashboards/TDinsight.md:356]: lgtm.

**Review by sangshuduo — APPROVED** [review]

**Pull request merged**